tests/cbmc/proofs/s2n_stuffer_alloc_ro_from_string/Makefile

```diff
@@ -0,0 +1,41 @@
+# Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may not use
+# this file except in compliance with the License. A copy of the License is
+# located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+# implied. See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Enough to get full coverage with 20 seconds of runtime.
+MAX_STRING_LEN = 100
```

Comment:
Is this just an upper bound to limit runtime?

Reply:
This is related to our verification approach: bounded model checking. CBMC will only unroll loops up to a certain bound (in this case, `strlen`) to build the model.